src/pid_design.jl

```diff
@@ -302,3 +302,50 @@ function loopshapingPI(P,ωp; ϕl=0,rl=0, phasemargin = 0, doplot = false)
     end
     return kp,ki,C
 end
+
+"""
+    piparams, C = placePI(P, ω₀, ζ; form=:standard)
+
+Selects the parameters of a PI-controller such that the poles of 
+closed loop between `P` and `C` are placed to match the poles of 
+`s^2 + 2ζω₀ + ω₀^2`.
+
+The form keyword allows you to choose which form the PI parameters
+should be returned on. 
+* `:standard` - `Kp*(1 + 1/Ti/s)`
+* `:series` - `Kc*(1 + 1/τi/s)`
+* `:parallel` - `Kp + Ki/s`
+* `:Ti` - `Kp + 1/(s*Ti)`
+
+`piparams` is a named tuple with the controller parameters.
+
+`C` is the transfer function of the controller.
+
+"""
+function placePI(P, ω₀, ζ; form=:standard)
+    num = numvec(P)[]
+    den = denvec(P)[]
+    length(den) == 2 || error("Can only place poles using PI if the system if of first order.")
```

Comment:
```suggestion
    length(den) == 2 || error("Can only place poles using PI if the system if of first order.")
```
```suggestion
    if length(den) != 2 || length(num) > 2
        error("Can only place poles using PI for proper first-order systems")
    end
```
Hmm, perhaps we should add something like this.

Reply:
I skipped that since I for some reason assume that `length(num)>length(den)`, but I guess that is if you want the system to be realisable and you can create systems which are not in the toolbox.